javascript/modules/client/src/internal/interfaces/plugins.ts

```diff
@@ -175,10 +194,58 @@ export type Erc20Proposal = Proposal & {
   open: boolean;
   executed: boolean;
 };
-
 export type Erc20Token = {
   address: string;
   name: string;
   symbol: string;
   decimals: number;
 };
+export interface IERC20ProposalQueryParams extends IPagination {
+  sortBy?: ERC20ProposalSortBy
+  addressOrEns?: string
+}
+
+
+export enum ERC20ProposalSortBy {
```

Comment:
There is places where `ERC20` is spelled in capitals and places where `Erc20` is spelled in camel case. So far, I would suggest always using camel case for readability

Reply:
This enum name was changed to `ProposalSortBy` so this does not apply anymore